lib/passageidentity/client.rb

```diff
@@ -58,6 +58,8 @@ def create_magic_link(
       ttl: 60,
       type: 'login'
     )
+      warn '[DEPRECATION] `create_magic_link()` is deprecated.  Please use `auth.create_magic_link()` instead.'
```

Comment:
it might be easier to delete this method's implementation and just make a call to `@auth.create_magic_link()` instead to avoid the duplication and maintenance for the minor release

Reply:
Will do with the gem implementation.